docs/apigateway.md

```diff
@@ -1,53 +1,100 @@
+**Note:** The APIs you created using the `wsk api-experimental` will continue to work for a short period, however you should begin migrating your APIs to web actions and reconfigure your existing apis using the new CLI command `wsk api`.
 
 ## OpenWhisk CLI configuration
-This experimental feature only works with the new OpenWhisk authentication model in which each namespace now has a unique authentication key associated with it.
-Follow the instructions in [Configure CLI](./README.md#setting-up-the-openwhisk-cli) on how to set the authentication key for your specific namespace.
-
-
-## Expose an OpenWhisk action
-
-Let's expose a simple action that is already pre-installed with OpenWhisk
-
-```
-$ wsk api-experimental create /hello /echo get /whisk.system/utils/echo
-```
-```
-ok: created api /echo GET for action /whisk.system/utils/echo
-https://21ef035.api-gw.mybluemix.net/hello/echo
-```
-A new URL is generated exposing the `echo` action via a __GET__ HTTP method.
 
-Let's give it a try by sending a HTTP request to the URL.
-```
-$ curl https://21ef035.api-gw.mybluemix.net/hello/echo?marco=polo
-```
-This will invoke the `echo` action, returning back a JSON string with the parameters sent.
-```
-{
-  "marco":"polo"
-}
-```
-
-You can pass parameters to the action via simple query parameters, or via request body.
+Follow the instructions in [Configure CLI](./README.md#setting-up-the-openwhisk-cli) on how to set the authentication key for your specific namespace.
 
-### Exposing multiple actions
+## Expose an OpenWhisk web action
+
+1. Create a JavaScript file with the following content. For this example, the file name is 'hello.js'.
+  ```javascript
+  function main({name:name='Serverless API'}) {
+      return {payload: `Hello world ${name}`};
+  }
+  ```
+  
+2. Create a web action from the following JavaScript function. For this example, the action is called 'hello'. Make sure to add the flag `--web true`
+  
+  ```
+  wsk action update hello hello.js --web true
+  ```
+  ```
+  ok: updated action hello
+  ```
+  
+3. Create an API with base path `/hello`, path `/world` and method `get` with response type `json`
+  
+  ```
+  wsk api create /hello /world get hello --response-type json
+  ```
+  ```
+  ok: created API /hello/world GET for action /_/hello
+  https://${APIHOST}:9001/api/21ef035/hello/world
+  ```
+  A new URL is generated exposing the `hello` action via a __GET__ HTTP method.
+  
+4. Let's give it a try by sending a HTTP request to the URL.
+  
+  ```
+  $ curl https://${APIHOST}:9001/api/21ef035/hello/world?name=OpenWhisk
+  ```
+  ```json
+  {
+  "payload": "Hello world OpenWhisk"
+  }
+  ```
+  The action `hello` got invoked, returning back a JSON string including the parameter `name` sent via query parameter. You can pass parameters to the action via simple query parameters, or via request body.
+  
+5. Full controll over the the http response
```

Comment:
`s/5./##/`
`s/controll/control`
`s/http/HTTP`

(This way we we can directly link to when answering questions…)